packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -117,11 +117,11 @@ Host tenants in a single org using Universal Directory (UD)
 
 ### Summary
 
-This configuration hosts all tenants in a single org. Customers have two
-ways to do this:
+This configuration hosts all tenants in a single org. Customers have one
+way to do this:
 
 * Roll your own custom solution
-* Use the [okta-dac](https://docs.idp.rocks/) project
+<!-- * Use the [okta-dac](https://docs.idp.rocks/) project-->
```

Comment:
Normally I'd say remove the bullet for a single item, but it looks like you're intentionally striking through these with the intent of adding it back later. Is that right?

Reply:
That is correct Graham. We will be adding the new Okta Dac replacement when it becomes available.